frontend/src/pages/EditStudent.vue

```diff
@@ -0,0 +1,309 @@
+<template>
```

Comment:
@szktm for this, it would be better if we can actually reuse the existing component that was created for New Student, instead of copy and pasting the code.

Reply:
Should I create a Student component that is used by both NewStudent and EditStudent to make it more readable? At any rate I note that this is not ready for merging - have marked it as a WIP for now.

Reply:
@szktm Yes, my suggestion would be to look at the existing NewStudent component and think about what it needs in order to work for EditStudent. Probably it should accept props, that if passed, prefill the fields?